plugin-templates/blocks/theme-navigation/index.scss

```diff
@@ -0,0 +1,5 @@
+/**
+ * Editor styles for the create-wordpress-plugin/theme-navigation block.
+ */
+
+.wp-block-create-wordpress-plugin-theme-navigation {}
```

Comment:
we can probably delete this file and the reference in block.json

Reply:
@mogmarsh would we then leave it to the developer to add these files in if needed?

Reply:
ah valid... let's leave it